docs/builders/index.mdx

```diff
@@ -13,37 +13,53 @@ multiple builders depending on the strategy you want to use to build the AMI.
 
 The Amazon plugin supports the following builders at the moment:
 
-- [amazon-ebs](/docs/builders/amazon/ebs.mdx) - Create EBS-backed AMIs by
+- [amazon-ebs](/docs/builders/amazon/ebs) - Create EBS-backed AMIs by
   launching a source AMI and re-packaging it into a new AMI after
   provisioning. If in doubt, use this builder, which is the easiest to get
   started with.
 
-- [amazon-instance](/docs/builders/amazon/instance.mdx) - Create
+- [amazon-instance](/docs/builders/amazon/instance) - Create
   instance-store AMIs by launching and provisioning a source instance, then
   rebundling it and uploading it to S3.
 
-- [amazon-chroot](/docs/builders/amazon/chroot.mdx) - Create EBS-backed AMIs
+- [amazon-chroot](/docs/builders/amazon/chroot) - Create EBS-backed AMIs
   from an existing EC2 instance by mounting the root device and using a
   [Chroot](https://en.wikipedia.org/wiki/Chroot) environment to provision
   that device. This is an **advanced builder and should not be used by
   newcomers**. However, it is also the fastest way to build an EBS-backed AMI
   since no new EC2 instance needs to be launched.
 
-- [amazon-ebssurrogate](/docs/builders/amazon/ebssurrogate.mdx) - Create EBS
+- [amazon-ebssurrogate](/docs/builders/amazon/ebssurrogate) - Create EBS
   -backed AMIs from scratch. Works similarly to the `chroot` builder but does
   not require running in AWS. This is an **advanced builder and should not be
   used by newcomers**.
 
 -> **Don't know which builder to use?** If in doubt, use the [amazon-ebs
-builder](/docs/builders/amazon/ebs.mdx). It is much easier to use and Amazon
+builder](/docs/builders/amazon/ebs). It is much easier to use and Amazon
 generally recommends EBS-backed images nowadays.
 
+## How to use this plugin
```

Comment:
Hmmm...part of me thinks that this information should only be on the overview page for each component type, as opposed to being on all documentation pages.

 I don't feel strongly about it to suggest a change but I find the duplication of the install instructions on each page  to be overwhelming.

Reply:
I also think is overwhelming 🤔  I added the info in the component page as well as the overview, for the same reason I added the authentication info to each of the data sources pages. 
When Megan was using the data sources she didn't think of looking in the overview page for more information, so I figure users will do the same. 
But even with that, I think I will remove it. It is a different situationn.

Reply:
left it only in the overview page when there's one.